omod/pom.xml

```diff
@@ -44,6 +44,18 @@
 			<classifier>tests</classifier>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>webservices.rest-omod-common</artifactId>
+			<version>2.21.0-SNAPSHOT</version>
+			<scope>provided</scope>
+		</dependency>
+                <dependency>
+                    <groupId>io.swagger</groupId>
```

Comment:
Shall we still need this if we use version 2.20.0?

Reply:
Version 2.20.0 works without the swagger methods 😊